src/app/static/src/app/store/surveyAndProgram/getters.ts

```diff
@@ -33,83 +8,4 @@ export const getters = {
     hasChanges: (state: SurveyAndProgramState) => {
         return !!state.survey || !!state.program || !!state.anc
     },
-
-    data: (state: SurveyAndProgramState) => {
-        const res = response(state);
-        return res ? res.data : null;
-    },
-
-    countryAreaFilterOption: (state: SurveyAndProgramState, getters: any, rootState: RootState): FilterOption => {
-        return rootState.baseline.shape!.filters!.regions as FilterOption;
-    },
-
-    filters: (state: SurveyAndProgramState, getters: any, rootState: RootState): DisplayFilter[] => {
```

Comment:
Found this and removed it as I realise we weren't using it anymore. I think we can also remove `selectedDataType` stuff everywhere, which will simplify a fair bit of the logic. I think this was used to open the same map in the input plots again to what they were currently viewing. But we have removed that and I don't really think it is worth the complexity. I will remove that in another PR to avoid noise here.